src/use_cases_execution/pto/Google-pto-script.js

```diff
@@ -167,3 +167,51 @@ function getNextWorkday(date) {
 
   return formatDateString(next);
 }
+// ---------- Notify ----------
+
+function notifyPendingStakeholders() {
+  const ss = SpreadsheetApp.getActiveSpreadsheet();
+  const sh = ss.getActiveSheet();
```

Comment:
Could we please adjust this one as well `sh` isn't descriptive at all

`ss` it's easier to understand but we could also use a more descriptive variable for clarity